commands/delegate_submission_test.go

```diff
@@ -0,0 +1,84 @@
+package commands_test
```

Comment:
Could you add test cases to prove the error message doesn't appear when data are valid for both delegation and undelegation ?

Reply:
I can and I have, thanks for the nudge.